treeherder/intermittents_commenter/commenter.py

```diff
@@ -201,12 +201,7 @@ def check_whiteboard_status(self, whiteboard):
         stockwell_text = re.search(r'\[stockwell (.+?)\]', whiteboard)
         if stockwell_text is not None:
             text = stockwell_text.group(1).split(':')[0]
-            if (
-                text == 'fixed'
-                or text == 'disable-recommended'
-                or text == 'infra'
-                or text == 'disabled'
-            ):
+            if text == 'fixed' or text == 'infra' or 'disable' in text:
```

Comment:
@jmaher Confirming there isn't some edge case that would make this change too broad.

Reply:
this should be just fine!